2-mandatory/4-space-colonies.js

```diff
@@ -15,7 +15,10 @@
   
 */
 
-function getSettlers() {}
+function getSettlers(arr) {
+ const willStay= arr.filter(string => string.includes("family") && string[0]==="A" );
```

Comment:
This is great! Can you find another way to check if a string starts with a particular letter?